src/org/gk/database/AttributeEditConfig.java

```diff
@@ -173,6 +210,23 @@ public void loadConfig(Document document) {
                 }
                 setUnedtiableAttNames(attNameList);
             }
+            else if (node.getNodeName().equals("uncreatableClass")) {
+            	NodeList list = node.getChildNodes();
+            	// get contact for class instances of uncreatableClasses.
+            	Element elmContact = (Element) node;
+            	String contact = elmContact.getAttribute("contact");
+            	setUncreatableClassContact(contact);
```

Comment:
Before this line, add: if (contact != null && contact.length() > 0) {} // To avoid assigning an empty contact.